src/modules/tfs/commentList/commentList.ts

```diff
@@ -9,8 +10,19 @@ import Comment from '../../../shared/comment';
  */
 export default class CommentList extends LightningElement {
   @api comments: Comment[];
+  @api user: User;
+
+  _id = 0;
 
   get showComments(): boolean {
     return this.comments.length > 0;
   }
+
+  /**
+   * Generate a unique key for the next comment
+   * A unique key must be generated so that the comment can be updated
+   */
+  get nextId() {
```

Comment:
This is really cool and much cleaner than a getter that maps an ID onto the array! But I don't know if it is good/bad? The docs say something about a "code pattern" for this but I could not find the pattern? https://lwc.dev/guide/html_templates#render-lists 

Curious where you got this method from?

Reply:
I came up with it myself. Originally, the comment key was the ID of the comment. I was having issues when I would replace the comment in the list because the old comment and the new comment had the same keys. Since they had the same keys, the new comment would not get re-rendered, so I decided to just keep a running count of incremental keys.

Reply:
Ok, sounds like it works good then.